infrastructure/.terraform.lock.hcl

```diff
@@ -2,22 +2,22 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/hashicorp/azurerm" {
-  version     = "4.5.0"
-  constraints = "~> 4.5.0"
+  version     = "4.8.0"
+  constraints = ">= 3.0.0, 4.8.0, < 5.0.0"
```

Comment:
here you can see it is constrained to these version from the module we call after terraform init